### ISSUE TITLE
Refuses to start

### ISSUE THREAD

**Issue by ndarilek** [jparyani/roundcubemail#3]
Seeing the following in my logs:

```
Debug log: Untitled Roundcube mailbox
...** SANDSTORM SUPERVISOR: Starting up grain.
*** Running /etc/rc.local...
*** Booting runit daemon...
*** Runit started as PID 16
[06-Nov-2015 06:01:24] NOTICE: [pool www] 'user' directive is ignored when FPM is not running as root
[06-Nov-2015 06:01:24] NOTICE: [pool www] 'group' directive is ignored when FPM is not running as root
nginx: [emerg] could not build the types_hash, you should increase either types_hash_max_size: 1024 or types_hash_bucket_size: 32
nginx: [emerg] could not build the types_hash, you should increase either types_hash_max_size: 1024 or types_hash_bucket_size: 32
nginx: [emerg] could not build the types_hash, you should increase either types_hash_max_size: 1024 or types_hash_bucket_size: 32
```

Thanks.

**Comment by jparyani** [comment]
Sorry, this was actually a known issue that was discussed on the sandstorm-dev list (see upstream niginx issue http://trac.nginx.org/nginx/ticket/352). I thought I had updated all my apps, but I guess had missed roundcube.

I'm going to attempt to move to vagrant-spk, which should fix this issue.

**Comment by jparyani** [comment]
Thanks for the report! Fixed in e537bf59271d85e530da174a5b0226e9606c425c and updated on the app market.
